storage/storage.go

```diff
@@ -1810,6 +1812,27 @@ func (o *ObjectHandle) Retryer(opts ...RetryOption) *ObjectHandle {
 	return &o2
 }
 
+// Retryer returns an Client that is configured with custom retry behavior as
+// specified by the options that are passed to it. All operations on the new
+// handle will use the customized retry configuration.
+// Retry options set on a bucket or object handle will take precedence over
+// these options.
```

Comment:
Maybe note that this should be configured before calling any network ops on the client. Otherwise there could be indeterminate behavior with operations in progress.